test/fluree/db/query/history_test.clj

```diff
@@ -680,93 +680,93 @@
+                          conn          @(fluree/connect
+                                          {:method       :file
+                                           :storage-path storage-path
+                                           :defaults
+                                           {:context      test-utils/default-context
+                                            :context-type :keyword
+                                            :did          (did/private->did-map
+                                                           test-utils/default-private-key)}})
+                          ledger        @(fluree/create conn ledger-name
+                                                        {:defaultContext
+                                                         ["" {:ex "http://example.org/ns/"}]})
+                          _             @(test-utils/transact ledger {:id   :ex/alice
+                                                                      :ex/x "foo-1"
+                                                                      :ex/y "bar-1"})
+                          _             @(test-utils/transact ledger {:id   :ex/alice
+                                                                      :ex/x "foo-2"
+                                                                      :ex/y "bar-2"})
+                          _             @(test-utils/transact ledger {:id   :ex/alice
+                                                                      :ex/x "foo-3"
+                                                                      :ex/y "bar-3"})
+                          _             @(test-utils/transact ledger {:id   :ex/cat
+                                                                      :ex/x "foo-cat"
+                                                                      :ex/y "bar-cat"})
+                          _             @(test-utils/transact ledger {:id   :ex/alice
+                                                                      :ex/x "foo-cat"
+                                                                      :ex/y "bar-cat"}
+                                                              {:message "meow"})
+                          loaded-ledger (test-utils/retry-load conn ledger-name 100)]
+                      (is (pred-match?
+                           [#:f{:assert  [{:ex/x "foo-3"
+                                           :ex/y "bar-3"
+                                           :id   :ex/alice}]
+                                :commit  {"https://www.w3.org/2018/credentials#issuer"
+                                          {:id test-utils/did?}
+                                          :f/address        test-utils/address?
+                                          :f/alias          ledger-name
+                                          :f/branch         "main"
+                                          :f/defaultContext {:id test-utils/context-id?}
```

Comment:
All of our files have a globally unique content-based hash (storage/location agnostic) @id, and also one or more :f/address where that file can be loaded from.

Not sure if defaultContext never had both of these, or if the :f/address got lost along the way, or maybe this isn't actually testing what physically exists in the file.

Just noticed it, so wanted to inquire to see if :f/defaultContext currently has that pattern, or if we need to create a ticket to get it updated.